docs/src/Popover.doc.js

```diff
@@ -1,261 +1,1316 @@
+                <Box marginEnd={4} marginStart={4}>
+                  <Flex direction="column" gap={8}>
+                    <List title="Top choices"/>
+                    <List title="All boards"/>
+                  </Flex>
+                </Box>
+              </Box>
+            </Box>
+          </Popover>
+        </Layer>
+      )}
+    </React.Fragment>
+  );
+}
+  `}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use Popover to communicate critical information, such as an error or interaction feedback. Instead, use the error supplied directly to the form element. See [related](#Related) to learn more."
+        defaultCode={`
+function PopoverExample() {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef();
+  const viewRef = React.useRef();
+
+  const isInViewport = () => {
+    const rect = viewRef && viewRef.current && viewRef.current.getBoundingClientRect();
+    const isVisible = rect
+      ? rect.top >= 0 &&
+        rect.left >= 0 &&
+        rect.bottom <= (window.innerHeight || document.documentElement.clientHeight) &&
+        rect.right <= (window.innerWidth || document.documentElement.clientWidth)
+      : undefined;
+
+    if (isVisible) {
+      setOpen(true)
+    }
+}
+
+  React.useEffect(() => {
+    isInViewport()
+    document.addEventListener('scroll',  isInViewport), [document, isInViewport]
+  });
+
+  return (
+    <React.Fragment>
+      <Box display="flex" alignItems="center" ref={viewRef} height={200}>
+        <TextField
+          id="field"
+          onChange={() => {}}
+          label="Name"
+          value="Cats"
+          ref={anchorRef}
         />
       </Box>
       {open &&
         <Layer>
           <Popover
             anchor={anchorRef.current}
-            idealDirection="up"
-            onDismiss={() => setOpen(false)}
+            color="red"
+            idealDirection="down"
+            showCaret
+            onDismiss={() => {}}
             positionRelativeToAnchor={false}
-            size="md"
+            size="xs"
           >
-            <Box padding={3} display="flex" alignItems="center" direction="column" column={12}>
-              <Text align="center">
-                Need help with something? Check out our Help Center.
+            <Box padding={3}>
+              <Text color="white" align="center">
+                You already have a board with that name
+              </Text>
+            </Box>
+          </Popover>
+        </Layer>}
+    </React.Fragment>
+  );
+}`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Popover to educate users on a new or existing feature. Be sure to use a blue background and include a caret pointing to the feature when Popover is triggered automatically, in the case of user education. See the [color and caret](#Color-and-caret) variant to learn more."
```

Comment:
Maybe: 

When Popover is triggered automatically, like in the case of user education, be sure to use a blue background and include a caret pointing to the feature.

Reply:
DONE